protocols/grpc/agent.proto

```diff
@@ -144,8 +144,9 @@ message Storage {
 	string driver = 1;           // empty in most cases. it will support "drbd", "bcache" ...
 	string source = 2;           // "/dev/sdb", "/dev/disk/by-scsi/xxxx", "none",...
 	string fstype = 3;           // "xfs", "ext4" etc. for block dev, or "9p" for shared fs, or  "tmpfs" for shared /dev/shm for all containers ...
-	repeated string options = 4; // options for the storage device, might required by some special devices.
-	string mount_point = 5;      // mount_point is only visible by VM, not by containers. This mount point can be used on oci.Mount.Source as "/Stroage/mount/point/{rootfs|_data}".
+	bool read_only = 4;          // specify if this should be mounted with the read only flag.
+	repeated string options = 5; // options for the storage device, might required by some special devices.
```

Comment:
a minor thing, how about move the `options` to the last

Reply:
@gnawux yes sure I can do that.